src/dtos/index.ts

```diff
@@ -11,6 +11,7 @@ import {
     ReplaceSmFunction,
     DeliverSmFunction,
     DeliverSmRespFunction,
+    DTOData,
```

Comment:
Add import to response, force only response events.
```suggestion
    DTOData,
    ResponseCommandName,
```